k8s-aws-elb-tagger.go

```diff
@@ -0,0 +1,286 @@
+package main
+
+import (
+	"expvar"
+	"flag"
+	"fmt"
+	"net"
+	"net/http"
+	"net/http/pprof"
+	"os"
+	"runtime/debug"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/inconshreveable/log15"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	// DefaultPort is the port the service is listening by default
+	DefaultPort = "3000"
```

Comment:
Just by curiosity, normal that we use PascalCase for these kinds of variables? Just wondering, it seems a bit out of place